cmd/clusterctl/yamlutil/BUILD.bazel

```diff
@@ -0,0 +1,24 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
```

Comment:
Why move this under `cmd/clusterctl` rather than leaving under `/util`? This might have other tooling uses outside of `clusterctl`

Reply:
The main reason was to avoid long term support for this package if they're moving outside of Cluster API. If there is a need, we can always move the package out but the only client today seems to be `clusterctl`. I can move it to another location as you suggested if you think it might be worth it